### ISSUE TITLE
Add upgrade to install mesa-steamos and lib32-mesa-steamos

### ISSUE THREAD

**Issue by LukeShortCloud** [LukeShortCloud/winesapOS#337]
This is REALLY complicated to automate as this conflicts with the default `mesa` and `lib32-mesa` packages that are installed in 3.0.0 and 3.0.1. Pacman has no built-in way to handle conflicts automatically. Work for trying to solve this has been started here: https://github.com/LukeShortCloud/winesapOS/issues/309 . We may just add this manual step to our upgrade notes.

**Comment by LukeShortCloud** [comment]
Upon further testing, the tricky part we ran into was building `mesa-steamos` and installing it with `yay`. There are a lot of prompts about if we (1) want to do a clean build, (2) modify the PKGBUILD, and (3) replace the conflicting package. Now that we have our own repository, we can just use `yes` to pipe to the installation command since it will only ask if we want to replace conflicting packages. Here is a simplified version of what we need to do:

```
yes | sudo pacman -S --noconfirm winesapos/mesa-steamos winesapos/lib32-mesa-steamos
```

**Comment by LukeShortCloud** [comment]
Resolved via: e61a1ed1182ce58b516120c6568aa24d9e1e2d1a (I accidentally put "Repalces" instead of "Resolves" in the commit message).
